### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by rhagarty**
- update readme with new instructions and videos
- remove yarn
- update packages

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,109 +69,171 @@ Use the ``Deploy to IBM Cloud`` button **OR** create the services and run locall
...
-![slack-bot-name](doc/source/images/slack-1.png)
+![get-creds](https://raw.githubusercontent.com/IBM/pattern-utils/master/watson-discovery/get-creds.png)
 
-Once created, save the **API Token** that is generated.
+  3. Copy and paste the `apikey` and `URL` values from your Watson Discovery service credentials into the .env file:
 
-![more-slack-config](doc/source/images/slack-2.png)
+```bash
+# Watson Discovery
+DISCOVERY_URL=&lt;add_discovery_url&gt;
```
Comment (markstur) [inline README.md]: the ampersand stuff is showing in the markdown render

File: README.md [hunk 2]
```diff
@@ -64,109 +69,171 @@ Use the ``Deploy to IBM Cloud`` button **OR** create the services and run locall
...
+### If you used the Deploy to IBM Cloud button...
 
-# Running the application
+If you used the **Deploy to IBM Cloud** option, most of the setup is automatic, but not quite all of it. We have to update a few environment variables.
+
+  1. From the IBM Cloud dashboard find the app that was created. Click on the **Runtime** tab and navigate to the **Environment variables** tab.
+
+  2. Add and set the Slack Bot **API Token** saved in the previous step.
+
+![environment-variables](doc/source/images/env_vars.png)
```
Comment (markstur) [inline README.md]: missing image (I think it is this one)

File: README.md (hunk 1 above)
Comment (rhagarty) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (rhagarty) [reply README.md]: done

**Comment by rhagarty** [comment]
will leave links in for now

File: README.md [hunk 3]
```diff
@@ -64,109 +69,171 @@ Use the ``Deploy to IBM Cloud`` button **OR** create the services and run locall
 ![toolchain-pipeline](doc/source/images/toolchain-pipeline.png)
 
 3. To see the app and services that were created use the [IBM Cloud dashboard](https://cloud.ibm.com). The app is named `watson-discovery-news` with a unique suffix. The following services are created:
```
Comment (markstur) [inline README.md]: maybe worth un-plural-ing "services" when you have a list of one

**Pull request merged**